src/data/relay_event_pii.txt

```diff
@@ -0,0 +1,46 @@
+Event.breadcrumbs.data
```

Comment:
Just a thought, you don't have to actually do this: You could make this file a `.json` file containing an array. That way you could just do `JSON.parse` and avoid the `split` & `trim` shenanigans.